### PR TITLE
feat: left menu review Display the space details panel - MEED-601 - Meeds-io/MIPs#16

### PR DESCRIPTION
**Pull request by SaraBoutej**
This change allows to display space detail panel in the left menu when clicking on space arrow  in the recent spaces portlet.

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue [hunk 1]
```diff
@@ -24,7 +25,7 @@
   </v-container>
 </template>
 <script>
-
+/* eslint-disable vue/one-component-per-file */
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue]: why need to disable it?

File: webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue [hunk 2]
```diff
@@ -172,11 +173,12 @@ export default {
         vuetify: Vue.prototype.vuetifyOptions,
       }).$mount(parentId);
     },
-    openOrCloseDrawer() {
+    openOrCloseDrawer(value) {
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue]:
```suggestion
    openOrCloseDrawer() {
```

File: webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue [hunk 3]
```diff
@@ -13,7 +14,7 @@
           {{ this.$t('menu.administration.title') }}
         </v-list-item-content>
         <v-list-item-action v-if="toggleArrow" class="my-0">
-          <v-btn icon @click="openOrCloseDrawer()">
+          <v-btn icon @click="openOrCloseDrawer(false)">
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue]:
```suggestion
          <v-btn icon @click="openOrCloseDrawer()">
```

File: webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/ExoHamburgerMenuNavigation.vue [hunk 4]
```diff
@@ -148,23 +176,30 @@ export default {
...
         return;
       }
       this.secondLevel = true;
+      this.thirdLevel = false;
       document.dispatchEvent(new CustomEvent('second-level-opened', {
         detail: {'contentDetail': contentDetail}
       }));
 
-      if (this.openedSecondLevel !== contentDetail.id && this.vueChildInstances[contentDetail.id].mountSecondLevel) {
+      if (this.openedSecondLevel !== contentDetail.id && this.vueChildInstances[contentDetail.id].mountSecondLevel || value) {
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/ExoHamburgerMenuNavigation.vue:189]: this value is always false

File: webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/ExoSpacesNavigationContent.vue [hunk 5]
```diff
@@ -166,7 +115,7 @@ export default {
   },
   methods: {
     searchSpaces() {
-      return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/spaces?q=&offset=${this.offset}&limit=${this.limitToFetch}&filterType=lastVisited&returnSize=true&expand=member,favorite`, {
+      return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/spaces?q=&offset=${this.offset}&limit=${this.limitToFetch}&filterType=lastVisited&returnSize=true&expand=member,managers,favorite`, {
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/ExoSpacesNavigationContent.vue:118]: should update preload url also

File: webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/SpaceNavigationItem.vue [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+    },
+    spaceAvatar() {
+      return this.space?.avatarUrl;
+    },
+    spaceDisplayName() {
+      return this.space?.displayName;
+    },
+    arrowIconClass() {
+      return this.arrowIcon;
+    },
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/SpaceNavigationItem.vue]:
```suggestion
```

File: webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/SpaceNavigationItem.vue [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+      <v-list-item-title class="body-2" v-text="spaceDisplayName" />
+    </v-list-item-content>
+    <v-list-item-icon
+      v-if="toggleArrow"
+      :disabled="loading"
+      :loading="loading"
+      class="me-2 align-center">
+      <v-btn icon @click="openOrCloseDrawer($event)">
+        <v-icon :id="space.id" class="me-0 pa-2 icon-default-color clickable" small>
+          {{ arrowIconClass }}
```
Comment (AzmiTouil) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/SpaceNavigationItem.vue]:
```suggestion
          {{ arrowIcon }} 
```

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/ExoHamburgerMenuNavigation.vue (hunk 4 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/ExoHamburgerMenuNavigation.vue:189]: no it's true from the recent space portlet.

File: webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/SpaceNavigationItem.vue (hunk 7 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/SpaceNavigationItem.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/SpaceNavigationItem.vue (hunk 6 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/SpaceNavigationItem.vue]: Done!

File: webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/ExoSpacesNavigationContent.vue (hunk 5 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/spaces-navigation/components/ExoSpacesNavigationContent.vue:118]: updated!

File: webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue (hunk 3 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue (hunk 2 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue (hunk 1 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue]: removed!

**Review by AzmiTouil — APPROVED** [review]

**Pull request merged**